app/controllers/policy/supported-policy-documents.js

```diff
@@ -6,17 +6,17 @@ const supportedPolicyDocuments = [
   {
     key: 'memorandum-of-understanding-for-crown-bodies',
     title: 'Pay memorandum of understanding',
-    template: 'policy/document/memorandum-of-understanding-for-crown-bodies'
+    template: 'policy/document/v2/memorandum-of-understanding-for-crown-bodies'
```

Comment:
We also need to upload new files to S3. Download is still pointing to old versions